Source/WebInspectorUI/UserInterface/Controllers/CallFrameTreeController.js

```diff
@@ -112,22 +162,23 @@ WI.CallFrameTreeController = class CallFrameTreeController extends WI.Object
 
     get treeOutline() { return this._treeOutline; }
 
-    get callFrames()
+    get stackTrace()
```

Comment:
Should this class really be `StackTraceTreeController` now? (Probably better as a followup if it should, though)

Reply:
yeah i was gonna have a followup to do that :)

Reply:
#2576